src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/OpenSslPkcs12Reader.cs

```diff
@@ -22,10 +22,7 @@ private OpenSslPkcs12Reader(SafePkcs12Handle pkcs12Handle)
 
         public unsafe static bool TryRead(byte[] data, out OpenSslPkcs12Reader pkcs12Reader)
```

Comment:
I've removed all the 'unsafe' declarations except one - it is still using pointers.